relay/metadata_provider.go

```diff
@@ -153,8 +155,15 @@ func (m *metadataProvider) fetchMetadata(key v2.BlobKey) (*blobMetadata, error)
 		}
 	}
 
+	blobSize := uint32(cert.BlobHeader.BlobCommitments.Length)
+	chunkSize, err := v2.GetChunkLength(cert.BlobHeader.BlobVersion, blobSize)
```

Comment:
Similarly, I think this returns the chunk length in number of symbols (not bytes). 
You would have to multiply this by `encoding.BYTES_PER_SYMBOL`

Reply:
done, thanks